src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -32,15 +32,32 @@ import Foundation
 struct KeyValueViewModel {
     let key: String
     let value: String
+    let valueNilable: Bool
```

Comment:
We should use consistent naming. Elsewhere we've called this `required`. The right Swift term would be `optional`. I'm okay with either.

Reply:
Renamed to optional.